aws_jobs/cyhy-data-extract.py

```diff
@@ -111,18 +113,23 @@ def cleanup_old_files(output_dir, file_retention_num_days):
 
 
 # TODO Finish function to delete files until there is only X in the bucket
-def cleanup_bucket_files(aws_access_key_id, aws_secret_access_key):
+def cleanup_bucket_files():
     """Delete oldest file if there are more than ten files in bucket_name."""
-    s3 = boto3.client(
-        "s3",
-        aws_access_key_id=aws_access_key_id,
-        aws_secret_access_key=aws_secret_access_key,
-    )
+    s3 = boto3.client("s3")
+
+    while True:
+        # Retrieve a list of applicable files.
+        ret = s3.list_objects_v2(Bucket=BUCKET_NAME, Prefix=SAVEFILE_PREFIX)
```

Comment:
This is nitpicky and feel free to ignore, but I don't love `ret` as the variable name here.  `response` (or `resp` if you really must abbreviate) will make this block more readable IMHO.

Reply:
I agree. I default to `ret`, but since this is a web query `response` or `resp` would correctly align with what's actually being done.